deployment/dockerdeploy/controller.go

```diff
@@ -543,16 +543,24 @@ func (c *Controller) UploadCertificates(
 	nodeCertPem []byte,
 	nodeKeyPem []byte,
 	caPems [][]byte,
+	nodeType string,
 ) error {
 	c.Logger.Debug("uploading couchbase certificates",
 		zap.String("container", containerID),
 		zap.Int("nodeCertLen", len(nodeCertPem)),
 		zap.Int("nodeKeyLen", len(nodeKeyPem)),
-		zap.Int("numCaPems", len(caPems)))
+		zap.Int("numCaPems", len(caPems)),
+		zap.String("nodeType", nodeType))
 
-	err := c.execCmd(ctx, containerID, []string{"mkdir", "-p", "/opt/couchbase/var/lib/couchbase/inbox/"})
+	installLocation := "couchbase"
+	if nodeType == "columnar-node" {
```

Comment:
Should this be analytics-node?

Reply:
There is no distinction between analytics and columnar in cbdino for the most part. nodeType for EA and colunnar is "columnar-node" for both.